lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -100,6 +100,16 @@ export class BrowserUtils {
         }
     }
 
+    /**
+     * Block redirectUri loaded in popup from calling AcquireToken APIs
+     */
+    static blockAcquireTokenInPopups(): void {
+        // Popups opened by msal popup APIs are given a name that starts with "msal."
+        if (window.opener && window.opener !== window && window.name.indexOf(`${BrowserConstants.POPUP_NAME_PREFIX}.`) === 0) {
```

Comment:
Might be good to check truthiness of `window.name` to be safe.
```suggestion
        if (window.opener && window.opener !== window && typeof window.name === "string" && window.name.indexOf(`${BrowserConstants.POPUP_NAME_PREFIX}.`) === 0) {
```